slack-base/src/main/java/com/hubspot/slack/client/models/blocks/objects/OptionGroupsResponseIF.java

```diff
@@ -0,0 +1,16 @@
+package com.hubspot.slack.client.models.blocks.objects;
```

Comment:
I think this model should live somewhere in this package https://github.com/HubSpot/slack-client/tree/master/slack-base/src/main/java/com/hubspot/slack/client/models/response